docs/manual/operator.md

```diff
@@ -230,4 +230,97 @@ the broker and addressing are running, and that these controllers have started s
 
 It is recommended that you deploy only a single instance of the ActiveMQ Artemis Operator in a given Kubernetes project. 
 Setting the replicas element of your Operator deployment to a value greater than 1, or deploying the Operator more than 
-once in the same project is not recommended.
\ No newline at end of file
+once in the same project is not recommended.
+
+## Configuring the Liveness and Readiness Probe
+
+The Liveness and readiness Probes are used by Kubernetes to detect when the Broker is started and to check it is still alive. 
+For full documentation on this topic refer to the [Configure Liveness, Readiness and Startup Probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) 
+chapter in the Kubernetes documentation.
+
+### The Liveness probe
+
+The Liveness probe is configured in the Artemis CR something like:
+
+```yaml
+spec:
+  deploymentPlan:
+    size: 1
+    image: placeholder
+    livenessProbe:
+      initialDelaySeconds: 5
+      periodSeconds: 5
+```
+
+If no Liveness probe is configured or the hndler itself is missing from a configured Liveness Probe then  the Operator 
+will create a default TCP Probe that will check the liveness of the broker by connecting to the web Server port, the default config is:
+
+```yaml
+spec:
+  deploymentPlan:
+    livenessProbe:
+      tcpSocket:
+        port: 8181
+      InitialDelaySeconds: 30,
```

Comment:
Should InitialDelaySeconds and TimeoutSeconds be initialDelaySeconds and timeoutSeconds (lowerCase)?

Reply:
yes I have fixed that